test/logic/player/music_player_test.dart

```diff
@@ -75,19 +84,23 @@ void main() {
       await binding.runAppTestWithoutUi(() async {
         MusicPlayer.instance.play();
         await binding.pump();
-        await MusicPlayer.handler!.onNotificationAction('stop');
+        await MusicPlayer.handler!.stop();
         await binding.pump();
         final playbackState = MusicPlayer.handler!.playbackState.value!;
         expect(playbackState.playing, false);
-        expect(playbackState.processingState, AudioProcessingState.ready);
```

Comment:
This has changed in `audio_service`, I verified that calling `stop` should cause an idle processing state.